Formula/lib/librist.rb

```diff
@@ -25,8 +24,12 @@ class Librist < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "cjson"
+  depends_on "libmicrohttpd"
   depends_on "mbedtls"
 
+  # Add build macos build patch
+  patch :DATA
```

Comment:
Is this issue and/or patch submitted upstream somewhere? If so, would be nice to link it here (can occur in a separate syntax-only PR to avoid re-spinning CI).

Reply:
I was trying to, but volan site is not accessible for me (I could not even login), I will just throw into the mailinglist later. With that being said, I think we can just merge for now.

Reply:
Yes, agree. Just wanted to check if it was being followed up on.

Reply:
just dropped an email into videolan@videolan.org